android/src/main/java/com/razorpay/flutter_customui/RazorpayPaymentActivity.java

```diff
@@ -81,6 +83,20 @@ private void createWebView() {
 
     private void sendRequest() {
         try {
+            if(payload.has(Constants.UPI_ACCOUNT)){
+                Log.d("RazorpayPaymentActivity", "payload "+payload);
+                String key  =  getAndRemoveKeyFromOptions(payload.getJSONObject("payload"));
+                razorpay.changeApiKey(key);
+                JSONObject newPayLoad =  payload.getJSONObject("payload");
+                newPayLoad.remove(Constants.KEY_ID);
+                UpiAccount upiAccount = RazorpayDelegate.getUpiAccount(payload.getString("upiAccount")) ;
+                HashMap<String, Object> payloadMap = new HashMap<>();
+                payloadMap.put("upiAccount", upiAccount);
+                payloadMap.put("payload", newPayLoad);
+                Log.d("RazorpayPaymentActivity", " payloadMap : "+ payloadMap +" "+ upiAccount.getAccountNumber());
```

Comment:
@AtifQamarRaz Can we remove the logs message? If needed, we can keep it for UAT.